.gitignore

```diff
@@ -53,3 +53,6 @@ atlassian-ide-plugin.xml
 
 # Crashlytics plugin (for Android Studio and IntelliJ)
 com_crashlytics_export_strings.xml
+
+# GitHub client id and client secret
+GithubConfig.java
```

Comment:
I would rather do this in some gradle settings not in some code exclusion.
E.g. credentials.gradle or credentials.properties.
Additional there should be a Section in some Document like the readme that explains how to configure the settings.

Excluding the file is fine, but an initial file with placeholders should be tracked inside the repository